### ISSUE TITLE
Launch app with Mac OS?

### ISSUE THREAD

**Issue by PimSpatial** [wleepang/DesktopDeployR#53]
First of all, thanks for this framework!

I want to make some ShinyApps accessible for my colleagues by using DesktopDeployR. I managed to use the framework to make apps launch from a shared folder and it works great. However, some colleagues use Mac instead of Windows. I am trying to make a separate launch file that would work for them. I have never worked with Macs before so my knowledge is limited.

Things I am aware off:
- Mac uses a .COMMAND file instead of a .bat file
- cscript / wscript are windows exclusive (not entirely sure)

Is there a way to make the framework work for Mac os? Does the .wsf file need to be replace with a Mac specific file?

Any help would be much appreciated!

Kind Regards,
Hybride

**Comment by wleepang** [comment]
@PimSpatial 

You are correct that wscript is exclusive to Windows. Unfortunately, I don't have access to a macOS device I can use to develop this framework.

Note, I originally developed this in an environment that did not allow use of containers, because at the time they were a relatively new technology.

Containers are now much more mature and accepted. If I were to revisit this framework today I would use a container instead of an R-portable installation. This has several benefits, like:
- better portability across OSes and CPU architectures
- better dependency management

Hope this helps.

**Comment by nev-awaken** [comment]
@PimSpatial 
You can take a look at this  https://github.com/eddelbuettel/rinside/tree/master/inst/examples/qt. If this works for you but I couldn't get it to run

If not then you have to wait until `webR` gets matured but there has been talks on it :  https://github.com/r-wasm/webr/issues/309

**Comment by PimSpatial** [comment]
@wleepang Thanks for the information! For now I am using the shinyapps.io environment, but when I need to develop something but more security I will for sure include your suggestion.

@nev-awaken Also much appreciated! It does look very promising so I will keep an eye out. It would be awesome if it gets more attention.
